<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureMeasureInitial</class>
 <widget class="QMainWindow" name="TemperatureMeasureInitial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>401</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>初始化参数</string>
    </property>
    <widget class="QLabel" name="Status">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>相机IP：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ROI">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>框选安全区域</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IP">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>192.168.0.122</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="LT_show">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>211</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Left_Top</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="RB_Show">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Right_Bottom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="template1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>370</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>选取模板</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="titleName">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>测温枪视觉检测参数初始化</string>
    </property>
   </widget>
   <widget class="QLabel" name="Image">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>相机图像</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="ImageShow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>511</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
